pkg/util/log.go

```diff
@@ -74,7 +74,8 @@ func (l LogAdapter) Println(v ...interface{}) {
 }
 
 // TODO(dannyk): remove once weaveworks/common updates to go-kit/log
-// 					-> we can then revert to using Level.Gokit
+//
+//	-> we can then revert to using Level.Gokit
```

Comment:
Can we omit the empty line?

```suggestion
// -> we can then revert to using Level.Gokit
```